source/CargoHold.cpp

```diff
@@ -135,7 +135,7 @@ void CargoHold::Save(DataWriter &out) const
 			}
 			firstOutfit = false;
 
-			out.Write(it.first->Name(), it.second);
+			out.Write(it.first->DisplayName(), it.second);
```

Comment:
`out.Write()` should always use the true name.

Reply:
done